js/scripts.js

```diff
@@ -0,0 +1,19 @@
+var url = 'http://api.icndb.com/jokes/random';
+var button = document.getElementById('get-joke');
+var paragraph = document.getElementById('joke');
+
+function getJoke() {
+	var xhr = new XMLHttpRequest();
+	xhr.open('GET', url);
+	xhr.addEventListener('load', function(){
+		var response = JSON.parse(xhr.response);
+		paragraph.innerHTML = response.value.joke;
```

Comment:
before you use joke to write it in HTML, you should check that value and joke exist in response ;)